README.rst

```diff
@@ -4,19 +4,17 @@ NuoDB Client Package
 
 .. contents::
 
-This project bundles the latest publicly-available versions of NuoDB_
-database clients into a single downloadable file.  The official versions of
-the NuoDB Client Package file are available both from `GitHub Releases`_ as
-well as from the `NuoDB Client Portal`_.
+The NuoDB Client drivers are used to connect Client applications to a NuoDB database. The NuoDB Client Package bundles the latest publicly-available NuoDB_ database client driver verions into a single downloadable file. The official versions of
+the NuoDB Client Package file are available from the NuoDB `GitHub Releases`_ page and from the NuoDB `Community Edition`_ webpage.
 
-You can use this project to create a package containing customized sets NuoDB
-Client packages, of course!
+You can use this project to create a package containing customized sets of NuoDB
+Client packages.
 
 Requirements
 ------------
 
-Each client will have different requirements.  System requirements for running
-the NuoDB database can be found on the `System Requirements`_ page.
+Each client application connecting to NuoDB database will have different requirements. System requirements for running
```

Comment:
_connecting to NuoDB database_ is wrong; maybe you mean _connection to a NuoDB database_

But this paragraph wasn't meant to discuss client applications (that is, customer applications).  It was meant to say that each client (as in the clients in the nuodb-client package) will have a different requirement for connecting to NuoDB.

Reply:
added line wrap and reverted sentence to original meaning.